.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/eng/SourceBuild*                                                                 @dotnet/source-build

```

Comment:
```suggestion

```

Reply:
An alternative is to leave this line and give write access to @dotnet/source-build , per this comment: https://github.com/dotnet/maintenance-packages/pull/140/files#r1819546289

Reply:
We don't have an eng/SourceBuild folder

Reply:
Oh, right. It was being added in the other PR.